app/models/elastic_search.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
```

Comment:
Is it possible to have unit or functional tests for this new functionality?

Reply:
yes.